hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/DataNode.java

```diff
@@ -2569,7 +2569,7 @@ public void shutdown() {
     if (this.threadGroup != null) {
       int sleepMs = 2;
       while (true) {
-        // When shutting down for restart, wait 2.5 seconds before forcing
+        // When shutting down for restart, wait 1 seconds before forcing
```

Comment:
nit
it should be ``1 second`` not ``1 seconds``

Reply:
Thank you for your reminder. Modified